File: HomeWork1/HomeWork1/Calculator.cs
namespace HomeWork1
{
    public class Calculator
    {
        public int Calculate(int val1,string operation, int val2)
        {
            var result = operation switch
            {
                "+" => val1 + val2,
                "-" => val1 - val2,
                "*" => val1 * val2,
                "/" => val1 / val2,
                _ => 0
            };
            return result;
            
        }
    }
}

File: HomeWork6/Giraffe11/Calculator.fs
module Giraffe11.Calculator

open System
open Giraffe11
open Parser

let inline Calculate val1 operation val2 =
    match operation with
    | CalculatorOperation.Plus -> val1 + val2
    | CalculatorOperation.Multiply -> val1 * val2
    | CalculatorOperation.Minus -> val1 - val2
    | _ -> val1 / val2

